src/utils.js

```diff
@@ -200,6 +194,23 @@ export const parseQRCode = (data) => {
   };
 };
 
+/**
+ * Extract a wallet address from a plain text string.
+ *
+ * @example
+ * 'hathor:<address>' -> '<address>'
+ *
+ * @example
+ * '<address>' -> '<address>'
+ *
+ * @param {string} plainText containing wallet address.
+ */
+function extractAddress(plainText) {
+  const [addressAlone, addressFromComposition] = plainText.split(':');
+  if (addressFromComposition) return addressFromComposition;
```

Comment:
I think we should keep the same validation we previously had in this case:

```
if (addressParts[0] !== 'hathor' || addressParts.length !== 2) {
    return {
      isValid: false,
      error: 'This QR code does not contain a Hathor address or payment request.',
    };
  }
```

Reply:
This piece of code removes the ability to accept a simple address, the one without the `hathor:` prefix. What I understood from the issue was that we want to allow this simple address to be recognized. Did I understand it wrong?

Reply:
No, you understood correctly. However, when we have a string with `:`, we must validate that the first part is `hathor` and we have only two parts.

Reply:
Got it!